scale-tests/spark-deploy-dispatchers.json

```diff
@@ -0,0 +1,37 @@
+{
+  "id": "spark-deploy-dispatchers",
+  "description": "Setup quota and dispatchers for Spark batch and streaming workloads",
+  "labels": {},
+  "run": {
+    "cpus": 1,
+    "mem": 1024,
+    "disk": 1024,
+    "cmd": "cd $MESOS_SANDBOX; git clone https://github.com/mesosphere/spark-build.git; cd spark-build/; git checkout $SPARK_BUILD_BRANCH; python3 -m venv test-env; pip3 install -r tests/requirements.txt; dcos cluster setup https://master.mesos --username=$DCOS_UID --password=$DCOS_PASSWORD --no-check; cd scale-tests/; python3 deploy-dispatchers.py --docker-image $DOCKER_IMAGE --quota-dispatcher-cpus $QUOTA_DISPATCHER_CPUS --quota-dispatcher-mem $QUOTA_DISPATCHER_MEM --quota-driver-cpus $QUOTA_DRIVER_CPUS --quota-driver-mem $QUOTA_DRIVER_MEM $NUM_DISPATCHERS $SERVICE_BASE_NAME $MESOS_SANDBOX/$OUTPUT_FILE",
+    "env": {
+      "DCOS_UID": "bootstrapuser",
+      "DCOS_PASSWORD": "deleteme",
+      "DOCKER_IMAGE": "mesosphere/spark-dev:f5dd540adffd9ab9e3e826e48d22e39ebc296567-aee120236ee37309892ddbee42acf0028096947f",
+      "SPARK_BUILD_BRANCH": "master",
+      "SERVICE_BASE_NAME": "dispatcher-mixed",
+      "NUM_DISPATCHERS": "1",
+      "OUTPUT_FILE": "dispatchers.out",
+      "QUOTA_DISPATCHER_CPUS": "1",
+      "QUOTA_DISPATCHER_MEM": "2048",
+      "QUOTA_DRIVER_CPUS": "4",
+      "QUOTA_DRIVER_MEM": "8192"
+    },
+    "placement": {
+      "constraints": []
+    },
+    "artifacts": [],
+    "maxLaunchDelay": 3600,
+    "docker": {
+      "image": "mesosphere/spark-build:latest"
```

Comment:
Is this image valid, or is it overridden in the python files?

Reply:
It's valid. It's the same image that is used in CI to run pytests. The ones that you may have seen in python files are images used by the Spark service itself (the image that runs Spark dispatchers and drivers).